src/commands/commit/withHook/index.js

```diff
@@ -26,4 +27,13 @@ export const registerHookInterruptionHandler = () => {
   })
 }
 
+export const cancelIfRebasing = (mode: 'hook' | 'client') =>
+  mode !== 'hook'
```

Comment:
Why not making this simpler and then switch the ternary order

```suggestion
  mode === 'hook'
```

Reply:
ok for me